mapproxy/config/loader.py

```diff
@@ -2060,7 +2113,26 @@ def demo_service(self, conf):
             image_formats=image_formats, srs=srs, services=services, restful_template=restful_template)
 
 
+def load_plugins():
+    """ Locate plugins that belong to the 'mapproxy' group and load them """
+    if sys.version_info >= (3, 8):
+        from importlib import metadata as importlib_metadata
+    else:
+        try:
+            import importlib_metadata
+        except ImportError:
```

Comment:
Also a good candidate for a logfile entry.

Reply:
done